cmd/exporter.go

```diff
@@ -44,9 +44,10 @@ const (
 )
 
 var (
-	address                      = flag.String("address", "0.0.0.0:8888", "bind address")
-	metricsPath                  = flag.String("metrics-path", "/metrics", "metrics path")
-	enableGPU                    = flag.Bool("enable-gpu", false, "whether enable gpu (need to have libnvidia-ml installed)")
+	address     = flag.String("address", "0.0.0.0:8888", "bind address")
+	metricsPath = flag.String("metrics-path", "/metrics", "metrics path")
+	enableGPU   = flag.Bool("enable-gpu", false, "whether enable gpu (need to have libnvidia-ml installed)")
+	// by default it should be "http://kepler-model-server.monitoring.cluster.local:8100/model" if we use model server
```

Comment:
the comment line changes the indent. Can you move it to the end of the next line?